MapboxNavigation/MapboxVoiceController.swift

```diff
@@ -221,7 +223,7 @@ open class MapboxVoiceController: RouteVoiceController, AVAudioPlayerDelegate {
         audioQueue.async { [weak self] in
             guard let `self` = self else { return }
             do {
-                self.audioPlayer = try AVAudioPlayer(data: data)
+                self.audioPlayer = try self.audioPlayerType.init(data: data)
```

Comment:
We can do this as follow-on work, but perhaps we should declare a type/protocol that we can also implement in a test double in order to verify data is sent through correctly